include/shell.h

```diff
@@ -178,6 +178,7 @@ class DOS_Shell : public Program, public HostShell {
 	void CMD_VER(char* args);
 	void CMD_LS(char* args);
 	void CMD_VOL(char* args);
+	void CMD_MOVE(char* args);
```

Comment:
Make it a pointer to a const string? (The signatures of the other commands should be the same but we gotta start somewhere)

Reply:
Ah okay, let's leave it then.